src/mainnet/target-assets/fortETH.sol

```diff
@@ -0,0 +1,462 @@
+        return totalAUM;
+    }
+
+    /// @dev Returns the maximum amount of the underlying asset that can be deposited into the Vault for the receiver, through a deposit call
+    function maxDeposit(address) public view override returns (uint256) {
+        uint256 _assetCap = convertToAssets(depositCap);
+        return _assetCap == 0 ? type(uint256).max : _assetCap - totalAUM;
+    }
+
+    /// @dev Returns the maximum amount of the Vault shares that can be minted for the receiver, through a mint call
+    function maxMint(address) public view override returns (uint256) {
+        return depositCap == 0 ? type(uint256).max : depositCap - totalSupply;
+    }
+
+    /********************************** Mutated Functions **********************************/
+
+    /// @dev Mints Vault shares to _receiver by depositing exact amount of underlying assets
+    /// @param _assets - The amount of assets to deposit
+    /// @param _receiver - The receiver of minted shares
+    /// @return _shares - The amount of shares minted
+    function deposit(uint256 _assets, address _receiver) external override nonReentrant returns (uint256 _shares) {
+        if (_assets >= maxDeposit(msg.sender)) revert InsufficientDepositCap();
+
+        _shares = previewDeposit(_assets);
+        
+        _deposit(msg.sender, _receiver, _assets, _shares);
+
+        IERC20(address(asset)).safeTransferFrom(msg.sender, address(this), _assets);
+        
+        _distributeAssets(_assets);
+
+        return _shares;
+    }
+
+    /// @dev Mints exact Vault shares to _receiver by depositing amount of underlying assets
+    /// @param _shares - The shares to receive
+    /// @param _receiver - The address of the receiver of shares
+    /// @return _assets - The amount of underlying assets received
+    function mint(uint256 _shares, address _receiver) external override nonReentrant returns (uint256 _assets) {
+        if (_shares >= maxMint(msg.sender)) revert InsufficientDepositCap();
+
+        _assets = previewMint(_shares);
+
+        _deposit(msg.sender, _receiver, _assets, _shares);
+
+        IERC20(address(asset)).safeTransferFrom(msg.sender, address(this), _assets);
+        
+        _distributeAssets(_assets);
+
+        return _assets;
+    }
+
+    /// @dev Distribute buffered WETH between AssetVaults
+    function distributeBufferedAssets(address _receiver, uint256 _minBounty) external nonReentrant {
+        uint256 bufferedBalance = ERC20(WETH).balanceOf(address(this));
+        if (!bufferedBalance>0) revert ZeroAmount();
+
+        uint256 _harvestBounty = _fees.harvestBountyPercentage;
+        if (_harvestBounty > 0) {
+                _harvestBounty = (_harvestBounty * bufferedBalance) / DENOMINATOR;
+                if (!(_harvestBounty >= _minBounty)) revert InsufficientAmountOut();
+                
+                IERC20(WETH).safeTransfer(_receiver, _harvestBounty);
+            }
```

Comment:
Nitpick, do we have some formatter or linter for Solidity? That indention looks off here.

Reply:
I'll do proper formatting.